modules/workflow-service-impl/src/main/java/org/opencastproject/workflow/impl/WorkflowServiceImpl.java

```diff
@@ -69,7 +69,6 @@
 import org.opencastproject.serviceregistry.api.ServiceRegistry;
 import org.opencastproject.serviceregistry.api.ServiceRegistryException;
 import org.opencastproject.serviceregistry.api.UndispatchableJobException;
-import org.opencastproject.util.Log;
```

Comment:
I haven't checked: Is this used anywhere else? Can this whole class be removed?

Reply:
It's still used, so you can't "just" remove it. Shouldn't be too hard though. But as Lars said, that's another PR.